test/source/tests/setup.ts

```diff
@@ -744,4 +746,47 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
 
   }
 
+  if (testVariant === 'CONSUMER-MOCK') {
+    ava.default('setup - imported key with multiple alias should show checkbox per alias', testWithBrowser(undefined, async (t, browser) => {
+      expect((await KeyUtil.parse(testConstants.keyMultiAliasedUser)).emails.length).to.equals(2);
+      const settingsPage = await BrowserRecipe.openSettingsLoginApprove(t, browser, 'multi.aliased.user@example.com');
+      await SetupPageRecipe.manualEnter(settingsPage, '', {
+        usedPgpBefore: false, submitPubkey: true, naked: true, fillOnly: true, checkEmailAliasIfPresent: true, key: {
+          title: 'multi.aliased.user@example.com',
+          passphrase: 'long enough to suit requirements',
+          armored: testConstants.keyMultiAliasedUser,
+          longid: null // tslint:disable-line:no-null-keyword
+        }
+      }, { isSavePassphraseChecked: false, isSavePassphraseHidden: false });
+      await settingsPage.close();
+    }));
+
+    ava.default('setup - imported key from a file with multiple alias', testWithBrowser(undefined, async (t, browser) => {
+      const settingsPage = await BrowserRecipe.openSettingsLoginApprove(t, browser, 'multi.aliased.user@example.com');
+      const key = {
+        title: 'unarmored OpenPGP key',
+        filePath: 'test/samples/openpgp/multialiaseduserexamplecom-0x357B908F62498DF8.key',
+        armored: null, // tslint:disable-line:no-null-keyword
+        passphrase: '1basic passphrase to use',
+        longid: null // tslint:disable-line:no-null-keyword
+      };
+      await SetupPageRecipe.manualEnter(settingsPage, key.title, { submitPubkey: true, usedPgpBefore: false, fillOnly: true, key });
+      expect(await settingsPage.isElementPresent('@container-for-import-key-email-alias')).to.equal(true);
+      expect(await settingsPage.isElementPresent('@input-email-alias-alias1examplecom')).to.equal(true);
+      expect(await settingsPage.isElementPresent('@input-email-alias-alias2examplecom')).to.equal(true);
+      /* simulate several clicks then exclude alias2@example.com from submitting key from the attester */
```

Comment:
Please add tests for `settingsPage.isChecked('@input-email-alias-alias1examplecom')`
and  `settingsPage.isChecked('@input-email-alias-alias2examplecom')`.
I expect alias2 to be checked by default.